File: src/CustomSDK/05_Examples/CSharp/CustomModules/CustomModule4CSharp/src/CustomModule.cs
namespace CustomModule4CSharp;

using System;
using CustomFramework4CSharp;

public class CustomModule : AbstractCustomModule
{
  public CustomModule() : base()
  {
  }

  public override void doSayHello()
  {
    System.Console.WriteLine("[CSharp]:CustomModule::Hello World");
  }
}

File: src/CustomSDK/Examples/Python/CustomModules/CustomModule4Python/src/CustomModule.cpp
#include "stdafx.h"

#include "CustomModule4Python/CustomModule.h"

#include <iostream>

CustomModule::CustomModule():
  AbstractCustomModule()
{
}

CustomModule::~CustomModule()
{
}

void CustomModule::doSayHello()
{
  std::cout << "[Python]:CustomModule::Hello World!" << std::endl;
}
